scapy/layers/l2.py

```diff
@@ -720,7 +719,7 @@ class Dot1AD(Dot1Q):
 conf.l2types.register(DLT_LINUX_SLL2, CookedLinuxV2)
 conf.l2types.register(DLT_ETHERNET_MPACKET, MPacketPreamble)
 conf.l2types.register_num2layer(DLT_LINUX_IRDA, CookedLinux)
-conf.l2types.register(DLT_LOOP, Loopback)
+conf.l2types.register(DLT_NULL, Loopback)
 conf.l2types.register_num2layer(DLT_NULL, Loopback)
```

Comment:
Doesn't it get redundant with this?
Should we have
`conf.l2typed.register_num2layer(DLT_LOOP, Loopback)` instead for compatibility?

Reply:
You are right.